docs/tables/aws_securityhub_insight.md

```diff
@@ -0,0 +1,57 @@
+# Table: aws_securityhub_insight
+
+An AWS Security Hub insight is a collection of related findings. It identifies a security area that requires attention and intervention. For example, an insight might point out EC2 instances that are the subject of findings that detect poor security practices. An insight brings together findings from across finding providers.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  insight_arn,
+  group_by_attribute,
+  region
+from
+  aws_securityhub_insight;
+```
+
+### List insights by a particular attribute
+
+```sql
+select
+  name,
+  insight_arn,
+  group_by_attribute,
+  region
+from
+  aws_securityhub_insight
+where
+  group_by_attribute = 'ResourceId';
+```
+
+### Get workflow status details for a particular insight
+
+```sql
+select
+  name,
+  insight_arn,
+  filters ->> 'WorkflowStatus' as workflow_status
+from
+  aws_securityhub_insight
+where
+  name = 'sp';
```

Comment:
What is the `sp` insight? It may be helpful to include it in the description, replacing `particular insight` with the name of the insight

Reply:
@cbruno10  `sp` is the name of a custom insight. I have updated the title of the query for better understanding.